packages/dapp-toolkit/src/modules/wallet-request/transport/connector-extension/connector-extension.module.ts

```diff
@@ -77,6 +77,15 @@ export const ConnectorExtensionModule = (input: {
       )
       .subscribe(),
   )
+  subscription.add(
+    subjects.responseSubject
+      .pipe(
+        mergeMap((walletResponse) =>
```

Comment:
how is this different from 
```
tap((walletResponse) => requestResolverModule.addWalletResponses([walletResponse])
```

Reply:
tap does not resolve/respect the underlaying promise